docs/audit/findings/audit/open/audit-6f4db61b.md

```
@@ -0,0 +1,78 @@
+    needs:
+      - Linter
+      - test
+    # Deliberately no `environment:`. Declaring one adds `:environment:<name>`
+    # to the OIDC token's subject claim, and npm's trusted publisher for this
+    # package was established from a publish.yml that had no environment at all
+    # (rekor logIndex 2429132485, commit 563f77a).
+```
+
+The signing certificate for the last published release binds an identity with no environment component:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)